### PR TITLE
add pelle

### PR DESCRIPTION
**Pull request by chimikoo**
Fatima and Akiko helped me! Thanks!

### CODE REVIEW

**Comment by carlotrimarchi** [comment]
@chimikoo add someone to the reviewers

**Review by akiko-luka — COMMENTED** [review]
Nicely done! :)

File: profile/README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 | Kirill | | |
 | Ezequiel | | |
 | Danyal | | |
-| Per-Emil|||
+| Per-Emil| Student |[@lazyshadow](https://github.com/chimikoo)|
```
Comment (FatimaBadaoui) [inline profile/README.md:12]: Well done!

**Review by FatimaBadaoui — APPROVED** [review]
Looks good!

**Pull request merged**